src/3p-frame.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-
+import './polyfills'; // Object.assign is used by serializeMessage
```

Comment:
Talked offline. Thanks @lannka for explaining how this work. This should not be an issue.